api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricsServiceLifecycle.java

```diff
@@ -255,6 +261,10 @@ public State getState() {
         return state;
     }
 
+    void eagerInit(@Observes @Initialized(ApplicationScoped.class) Object event) {
+        // init
+    }
+
```

Comment:
That's good to know.  You may want to add that as an inline comment if you end up going back into this file.